pkg/controller/operator/defaults/defaults.go

```diff
@@ -682,6 +712,45 @@ func defaultVersioning(settings *kubermaticv1.KubermaticVersioningConfiguration,
 	return nil
 }
 
+func defaultExternalClusterVersioning(settings, defaults map[kubermaticv1.ExternalClusterProviderType]kubermaticv1.ExternalClusterProviderVersioningConfiguration) error {
+	// this should never happen as the resources are not pointers in a KubermaticConfiguration
+	if settings == nil {
+		settings = make(map[kubermaticv1.ExternalClusterProviderType]kubermaticv1.ExternalClusterProviderVersioningConfiguration)
+	}
+
+	if len(settings) == 0 {
+		settings = defaults
+	}
+
+	eksProviderVersioningConfiguration, ok := settings[kubermaticv1.EKSProviderType]
+	if !ok {
+		settings[kubermaticv1.EKSProviderType] = defaults[kubermaticv1.EKSProviderType]
+	} else {
+		if len(eksProviderVersioningConfiguration.Versions) == 0 {
+			eksProviderVersioningConfiguration.Versions = defaults[kubermaticv1.EKSProviderType].Versions
+		}
+		if eksProviderVersioningConfiguration.Default == nil {
+			eksProviderVersioningConfiguration.Default = defaults[kubermaticv1.EKSProviderType].Default
+		}
+	}
+	settings[kubermaticv1.EKSProviderType] = eksProviderVersioningConfiguration
+
+	aksProviderVersioningConfiguration, ok = settings[kubermaticv1.AKSProviderType]
```

Comment:
```suggestion
	aksProviderVersioningConfiguration, ok := settings[kubermaticv1.AKSProviderType]
```

This might be the issue.

Reply:
I tested locally after removing t.Parallel(), and the tests passed. should I go ahead and push the tests without t.Parallel()